plugins/uberdev/skills/merge/SKILL.md

```
@@ -559,6 +559,28 @@ Per-PR detail block (one per PR in the run):
+      - file: <relative path>
+        verdict: [refused] | [ambiguous]             (lowercase, ∈ PARK_REASON_ENUM)
+        justification: <sanitize_agent_text(resolution_summary) | fmt -w 80>
+        risks:                                       (only if agent return's risks[] is non-empty)
+          - <sanitize_agent_text(risks[0]) | fmt -w 80>
+          - <...>
 ```
 
+- **Conditional render.** The `conflict files:` sub-block appears ONLY when `outcome` is `Parked` AND `park reason` is `refused` or `ambiguous`. For `test-fail-exhausted` and `push-non-ff`, the sub-block is omitted (those park reasons have no per-file conflict context).
+- **Field source mapping.** `file` ← agent input `file_path`. `verdict` ← agent return `status`, lowercased and bracketed. `justification` ← agent return `resolution_summary`. `risks` ← agent return `risks[]`.
+- **Sanitization.** Each text field passes through `sanitize_agent_text` before display:
+  ```sh
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)